tests/Paket.Tests/ProjectFile/TestData/paket.dependencies

```diff
@@ -0,0 +1 @@
+#empty file to stop paket.install from messing with files here
```

Comment:
Good idea. I always reverted/not commited the test folder tbh...

We should make a list what we like to do for breaking changes, just so we can take a look at this list for every major.
Changing this behavior is definitely one of them...